packages/flutter/test/services/asset_bundle_test.dart

```diff
@@ -83,4 +83,21 @@ void main() {
 
     expect(bundle.toString(), 'NetworkAssetBundle#${shortHash(bundle)}($uri)');
   }, skip: isBrowser); // https://github.com/flutter/flutter/issues/39998
+
+  test('Throws expected exceptions when loading not exists asset', () async {
+    TestWidgetsFlutterBinding.ensureInitialized();
```

Comment:
One strange thing I found during tests is if we call the `ensureInitialized` in this test file, the "NetworkAssetBundle control test" will fail because the request returns `400` instead of `404`.

Reply:
the widget tester overrides HTTP calls to always return 400.

Reply:
> the widget tester overrides HTTP calls to always return 400.

TIL.

Reply:
Eventually I update the test to use `400` instead.